recipe/meta.yaml

```diff
@@ -80,7 +89,6 @@ outputs:
         - numpy >=1.24,<3
         - pywin32                         # [win64]
         - cupy                            # [cuda_compiler_version != "None"]  # TODO: remove me in the next release
```

Comment:
Q: I thought this can be removed too?

Reply:
We still rely on cupy for our array implementation, so for the GPU build of nvmath, we need the end user to have cupy available? Unless you're thinking that the end user could just use pytorch or numpy arrays instead.

Reply:
I see what you mean, alright. Then perhaps we should remove the TODO comment?

Reply:
I checked the source code for v0.2.0, we still have a hard dependency on CuPy for FFT execution using the CUDA backend. Thus, the CUDA variants of the base package still have a hard dependency on CuPy.

```
        try:
            import cupy
        except ImportError as e:
            raise RuntimeError(
                "Currently, the FFT CUDA execution requires cupy. " "Please make sure cupy is installed."
            ) from e
```

Users who are using CPU execution space-only do not need CuPy now since all imports of CuPy are now guarded with try-except. So removing that dep from the non-CUDA variant is OK.